dags/common/pull_ftp.py

```diff
@@ -6,28 +6,85 @@
 from airflow.api.common import trigger_dag
 from common.repository import IRepository
 from common.sftp_service import SFTPService
+from structlog import PrintLogger
 
 
-def migrate_from_ftp(sftp: SFTPService, repo: IRepository):
-    filenames = sftp.list_files()
+def migrate_files(filenames, sftp: SFTPService, repo: IRepository, logger: PrintLogger):
```

Comment:
For me this function, looks pretty long, what do you think about splitting it into smaller functions (move them to utils) and using these functions here? Would be easy readable